src/App.jsx

```diff
@@ -1,35 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from 'react';
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  {
+    id: 1,
+    title: "Commit project and create working branch"
+  },
+  {
+    id: 2,
+    title: "Create the beginnings of the todo list"
+  },
+  {
+    id: 3,
+    title: "Complete assignment"
+  }
+];
+
+function listItem (item) {
+  return (
+    <li key={item.id}>
+      <span>{item.id} </span>
+      <span>{item.title}</span>
+    </li>
+ )
+}
 
+function App() {
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <div>
+      <h1>Todo List</h1>
+      <hr /> 
+
+      <ul>
+      {todoList.map(listItem)}   
```

Comment:
Something to flag as we move to future lessons -  you'll learn that what's going on here is you are returning a React component, so best practice is to  tweak the way this mapping is written out. No need to worry about it now since this way also technically works, but we'll get to it in an upcoming lesson.